### DISCUSSION TITLE
Can't apply custom changes

### DISCUSSION THREAD

**Discussion by EliaquimNascimento** [ankitskvmdam/clean-jsdoc-theme#146] [General]
Edit: Well, I woke up in the morning, and the issue was solved. Don't know why. Maybe it is something related to the Live server extension of the VS code not reloading properly.
But everything works fine now ^^

Hi everyone.
I'm new to this and have some trouble changing the default theme or applying any new settings under the _"theme_opts"_ on my jsdoc.json file. Here is the code:
`{
    "plugins": ["plugins/markdown"],
    "recurseDepth": 10,
    "source": {
        "include": ["MakerDocs", "package.json"],
        "includePattern": ".+\\.js(doc|x)?$",
        "excludePattern": "(node_modules/|docs)"
    },
    "sourceType": "module",
    "tags": {
        "allowUnknownTags": true,
        "dictionaries": ["jsdoc","closure"]
    },
    "templates": {
        "cleverLinks": false,
        "monospaceLinks": false
    },
    "opts": {
        "encoding": "utf8",
        "readme": "./README.md",
        "destination": "docs/",
        "recurse": true,
        "verbose": true,
        "template": "./node_modules/clean-jsdoc-theme",
        "theme_opts": {
            "default_theme": "dark",
            "search": false
        }
    },
        "markdown": {
        "hardwrap": false,
        "idInHeadings": true
    }
}`

I set it to light, but it is always dark.
I also tried other things, like removing the search, and the changes didn't apply either to my docs.

I followed the README and I also didn't manage to generate the docs using _npm run generate-docs_, despite having it as a script on my package.json. But it does generate the docs if I just use the plain script: _"generate-docs": "jsdoc --configure jsdoc.json --verbose"_ on the terminal.

This is my package.json:
`{
  "compilerOptions": {
    "module": "commonjs",
    "target": "es6"
  },
  "dependencies": {
    "clean-jsdoc-theme": "^4.1.7"
  },
  "name": "js",
  "version": "1.0.0",
  "main": "main.js",
  "directories": {
    "doc": "docs"
  },
  "devDependencies": {},
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
    "generate-docs": "jsdoc --configure jsdoc.json --verbose"
  },
  "repository": {
    "type": "git",
    "URL": "....."
  },
  "keywords": [],
  "author": "",
  "license": "ISC",
  "bugs": {
    "URL": "....."
  },
  "homepage": "....",
  "description": ""
}`

So I guess what can help me is find a way to confirm that my workspace is really using that package.json and my jsdoc is really using that jsdoc.json file as a configuration JSON.
Can anyone point me in some direction on how can I check this?

EDIT: I fixed some errors on my package and jsdoc JSON files. And I manage to make the _npm run generate-docs_ work. The problem with, I believe it is a typo on the readme file:
Instead of `"script": ...` it must be `"scripts": ...`.

But still didn't manage to apply changes with custom options.

**Comment by ankitskvmdam** [comment]
Hi @EliaquimNascimento
For some reason, I didn't receive the notification for this.
I am really happy that it is working for you.

Thanks for also pointing out the typos, I have fixed that one as well!

Feel free to raise an issue or start a new discussion whenever you want.

**Comment by ankitskvmdam** [comment]
v5 supports custom CSS and JS (`customCss`/`customJs` and the `customCssFile`/`customJsFile` variants), plus component overrides. Docs: https://ankdev.me/clean-jsdoc-theme/theme/configuration/ Closing as resolved.
